commands2/typing.py

```diff
@@ -0,0 +1,7 @@
+from typing import TypeVar
+
+from wpimath.controller import ProfiledPIDController, ProfiledPIDControllerRadians
+
+GenericProfiledPIDController = TypeVar(
+    "GenericProfiledPIDController", ProfiledPIDControllerRadians, ProfiledPIDController
```

Comment:
Oh neat, this works? This definitely seems like the Right Way to do this, way simpler than a Protocol